### PR TITLE
Lab 4, 5 and 6 Completed - Varun B Majithia (Student ID - 8894799)

### PR DESCRIPTION
**Pull request by Varunmajithia**
Lab4 html link is https://varunmajithia.github.io/CSCN8010-lab-submissions/8894799/Lab4.html
Lab5 html link is https://varunmajithia.github.io/CSCN8010-lab-submissions/8894799/lab5.html
Lab 6 html link is https://varunmajithia.github.io/CSCN8010-lab-submissions/8894799/lab6.html
Respected Professor,
I have submitted lab 4, lab 5 and lab 6.
Above are the html links for the same.
Please let me know if there are any improvisations needed.
Thank you.

Best Regards
Varun B Majithia

### CODE REVIEW

**Review by RanFeldesh — APPROVED** [review]
Lab 4: 6/6 points

Lab 5: 5.5/6 points STD is missing per model, this allows model comparison to a better degree

Lab 6: 5.5/6 points, good submission and good analysis. NOte that class imbalance can't explain a perfect accuracy. overfitting is a case where training data is much higher than testing, here both are high. But the argument of a small number of data samples is very relevant. A larger set would have probably lowered the accuracy.
